models/models.js

```diff
@@ -6,35 +6,52 @@ const readTopics = () => {
   });
 };
 
-const readArticles = () => {
-  return db
-    .query(
-      `SELECT articles.article_id, articles.article_img_url, articles.author, articles.created_at, articles.title, articles.votes, articles.topic, COUNT(comments.article_id) AS comment_count
-      FROM articles
-      LEFT JOIN comments on comments.article_id = articles.article_id
-      GROUP BY articles.article_id
-      ORDER BY articles.created_at DESC;`
-    )
-    .then((articles) => {
-      return articles.rows;
-    });
-};
+const readArticles = (query) => {
```

Comment:
Would you consider passing the sort_by and the order through the parameters? Considering that they are variables that change dependent on the input of the client?